wiki/About/de.md

```diff
@@ -0,0 +1,5 @@
+# Über
```

Comment:
Sieht deutschsprachig bisschen out-of-context aus. Vielleicht einfach den Kontext dahinterpacken?